src/Disbursements.php

```diff
@@ -79,7 +86,30 @@ public static function createBatch($params = [], $options = [])
      * @param string $external_id external id
      * @param array  $options     user's options
      *
-     * @return array
+     * @return array[
+     *  [
+     * 'user_id'=> '5785e6334d7b410667d355c4',
```

Comment:
Why this one returns the actual value, not the data type?

Reply:
Get disbursements by external ID returns an array of disbursements, so I'm writing that to show a sample of response.

Reply:
So you are writing the docblock with example value for functions that return an array of a certain item?

Reply:
Yes